src/main/java/ConsoleCharacterRepository.java

```diff
@@ -5,37 +5,41 @@
 public class ConsoleCharacterRepository {
 
 
-//    List<Integer> charactersWeightsList = new ArrayList<>();
-//    Scanner weightsInput = new Scanner(System.in);
-
     public List<Character> getCharacters() {
+
         List<Character> charactersNamesList = new ArrayList<>();
         Scanner namesInput = new Scanner(System.in);
 
         System.out.println("Type character names from which you want to roll:");
 
         while (true) {
 
-            String characterName = namesInput.nextLine();
+            String characterData = namesInput.nextLine();
```

Comment:
Dodawanie `Data` do nazw nic nie wnosi, bo to słowo można dać do wszystkie i będzie to wyglądało "ok" bo przecież w programowaniu wszystko jest danymi.

Fajnie o tym mówi Sławek tutaj https://youtu.be/0uGzB8xp2ag?t=2703 w innym kontekście ale łatwo przełożyć, polecam zobaczyć parę minut. O Teorii informacji Shannona.

Nazwałbym to `unparsedCharacter`. Bo to są informacje o postaci ale jeszcze nieprzetworzone w sposób przyjazny dla naszego systemu.